src/bp/core/services/converse.ts

```diff
@@ -68,8 +68,14 @@ export class ConverseService {
       payload.type = 'text'
     }
 
-    if (payload.type === 'text' && (!payload.text || !_.isString(payload.text) || payload.text.length > 360)) {
-      throw new InvalidParameterError('Text must be a valid string of less than 360 chars')
+    let apiTextLimitation = _.get(await this.configProvider.getBotConfig(botId), 'converse.maxMessageLength')
+
+    if (!apiTextLimitation) {
+      apiTextLimitation = _.get(await this.configProvider.getBotpressConfig(), 'converse.maxMessageLength', '360')
```

Comment:
Can you change 360 from a string to a number ?. Also just a detail, but you could also use maxMessageLength as the variable name